### ISSUE TITLE
Further explanation of setup

### ISSUE THREAD

**Issue by bschatzow** [basschipper/homeassistant-generic-hygrostat#36]
In your readme you have examples of a binary sensor with default settings  :

    attribute: humidity # Optional use sensor attribute instead of state.

    delta_trigger: 3 # Optional humidity swing to detect. Default = 3
What is allowed?  Is this a swing from the current value to +3. i.e 50 to 53?

    target_offset: 3 # Optional dehumidification target offset. Default = 3
What is allowed?  Is this a swing from the current value to +3. i.e 50 to 53?

    min_on_time: 300 # Optional min on time in seconds. Default = 0 seconds
I assume this is the time when the sensor first turns on.  The 300 means it will stay on for 5 minutes?
    max_on_time: 7200 # Optional safety max on time in seconds. Default = 7200 seconds
This will shut it off in 2 hours?
    sample_interval: 300 # Optional time between taking humidity samples in seconds, default 300 seconds
This takes a sample every 5 minutes?  This seems long for a shower.
    min_humidity: 30 # Optional minimum humidity to enable dehumidification. Default = 0
This means that it will not turn on below 30?

Are multiple Hygrostat binary sensors setup by using a second, third, etc binary sensor in the yaml file?

My sensor file:
binary_sensor:
  - platform: generic_hygrostat
    name: Bathroom Hygrostat
    sensor: sensor.bill_br_govee5075_humidity # Source humidity sensor
    delta_trigger: 3 # Optional humidity swing to detect. Default = 3
    target_offset: 3 # Optional dehumidification target offset. Default = 3
    min_on_time: 300 # Optional min on time in seconds. Default = 0 seconds
    max_on_time: 7200 # Optional safety max on time in seconds. Default = 7200 seconds
    sample_interval: 300 # Optional time between taking humidity samples in seconds, default 300 seconds
    min_humidity: 30 # Optional minimum humidity to enable dehumidification. Default = 0
    unique_id: bill_bathroom_hygrostat # Optional ID that uniquely identifies this sensor. Set this to a unique value to allow customization through the UI.

This is what the sensor looks like in developers tools.
![image](https://user-images.githubusercontent.com/36886571/174274385-f457ffbf-e447-4c1d-9c52-ea8e57bb3958.png)
Not sure why it shows null for target?

**Comment by basschipper** [comment]
You're asking a lot of questions that are not so much related to this custom integration, but more about standard Home Assistant behavior. Like the sensor state vs state attr. https://www.home-assistant.io/docs/configuration/templating/. Or like adding multiple sensors. All default HA stuff.

From the looks of it it is working just fine, obviously there is no target at the moment because the state is `off`. As soon as a swing is detected the state will go to `on` and the attr (`target`, `min_on_timer`, `max_on_timer`) will be set.

Also just start simple and don't try to use all optional settings right away, they are optional for a reason.

```yaml
binary_sensor:
  - platform: generic_hygrostat
    unique_id: bill_bathroom_hygrostat 
    name: Bathroom Hygrostat  
    sensor: sensor.bill_br_govee5075_humidity
# Second hygrostat
  - platform: generic_hygrostat
    unique_id: bill_swimming_pool_hygrostat 
    name: Swimming Pool Hygrostat  
    sensor: sensor.bill_br_govee5075_humidity_2
```

Restart HA, then give it some time (~20 min) to collect some samples. And then increase the humidity for >5 min and see if the state goes to `on` and the attr are set.

**Comment by bschatzow** [comment]
Thanks, I understood the multiple sensors, but it was not clear as some applications require sperate installs for multiple sensors ( Waze for example). This app is easy. 
I did not understand why the target value is not set after capturing sensor readings. I thought I would see a target of 3 above my reading. 
I also did not understand what values are valid. You show a default but no explanation of range allowed or if it is a percentage or  the actual number.

**Comment by bschatzow** [comment]
After humidity was added I got this:
![image](https://user-images.githubusercontent.com/36886571/174358652-106b82cf-a104-4d42-b7e2-ef5287351df0.png)

Did not trigger the automation until above 75.
In a small room like a bathroom the humidity rises very quickly with a hot shower on.  Is there anyway to get it to trigger faster?

**Comment by ADNovo** [comment]
What is the downside of lowering sample_interval?

**Comment by bschatzow** [comment]
My understand (not sure if I am correct) is that it can increase system load. I have not seen an issue. I lowered mine as I wanted it to trigger the fan faster.

**Comment by ADNovo** [comment]
How low did you take it? 1 min? 2 min?

**Comment by bschatzow** [comment]
I am currently under a minute (50 seconds). Since you asked, I am going to bring it down to 10 seconds and see what happens.  I believe I may see false triggers?  I'll let you know.  I have tried many different ways to turn on the fan and have not been happy with any of them.  Either I am not understanding correctly how to implement them or something else is causing my issue.  

When I set an above a certain value, it triggers very fast.  The issue for me is the above value is always changing due to AC, weather etc.  I have tried different ways to trigger , trend lines,  Ave vs threshold and others and they all seem like they should be valid solutions, but they all trigger much slower than just setting a value.  Not sure why?
